plugins/accelerated-peft/README.md

```diff
@@ -19,9 +19,18 @@ Plugin | Description | Depends | Loading | Augmentation | Callbacks
 ## Known Issues
 
 - Models with sliding windows (e.g., Mistral, Mixtral) will have [memory and throughout issues](https://github.com/huggingface/transformers/issues/30461).
-- GPTQ-LORA sometimes observed to have `nan` grad norms in the begining of training, but training proceeds well otherwise.
+- GPTQ-LORA sometimes observed to have `nan` grad norms in the begining of training, but training pr.
 - `low_cpu_mem_usage` temporarily disabled for AutoGPTQ until bug with `make_sure_no_tensor_in_meta_device` is resolved.
-- Requires nightly [AutoGPTQ](https://github.com/AutoGPTQ/AutoGPTQ) until package `> 0.7.1` becomes available
+- The legacy implementation of GPTQ-LORA uses an external AutoGPTQ package, you must ensure the package is installed
```

Comment:
Lets not have this under Known Issues. Make a proper section in the README and explain
- the GPTQ depends on AutoGTPQ
- there are two options, using the package we extracted from Model Cloud
- or use the external lib
- external lib have to make sure to use this commit.